library/src/main/java/io/ona/kujaku/services/MapboxOfflineDownloaderService.java

```diff
@@ -274,13 +318,30 @@ private void sendBroadcast(SERVICE_ACTION_RESULT serviceActionResult, String map
     }
 
     /**
-     * Sends a local broadcast with the result of a service operation & mapName except for {@code Constants.SERVICE_ACTION.DELETE_MAP}
+     *Sends a local broadcast with the result of a service operation & mapName. To capture the
```

Comment:
Is there supposed to be a space between the * and S?

Reply:
Yea